src/foam/nanos/column/NestedPropertiesExpression.js

```diff
@@ -37,8 +37,32 @@ foam.CLASS({
     }
   ],
   methods: [
-    function set(o, val) {
-      o.cls_.getAxiomByName(this.nestedProperty).set(o, val);
+    {
```

Comment:
should we maybe add void set method to AbstractExpr ? or create some kind of PropertyBasedExpression and add method there?

Reply:
I don't think so. You wouldn't normally think of set() as being part of an Expression. Better to just catch ClassCastException or check if InstanceOf to make code safe. Or Make a Settable or Setter interface or SettableExpression.